### DISCUSSION TITLE
react-native-pager-view new architecture adoption

### DISCUSSION THREAD

**Discussion by thymikee** [reactwg/react-native-new-architecture#16] [Libraries]
@troZee is working on converting https://github.com/callstack/react-native-pager-view to Fabric. @cortinico mind adding him write access to the discussion?

**Comment by cipolleschi** [comment]
Hi @thymikee @troZee: what's the current status of the migration? Are there blockers? Do you have an ETA?

**Reply by thymikee** [reply]
@troZee implemented a PoC for iOS and next week he'll asses if this is something that he'll be able to continue in the next months or pushed to Q4

**Comment by thymikee** [comment]
Fabric support for both platforms is released in https://github.com/callstack/react-native-pager-view/releases/tag/v6.1.0-rc.0
